src/explore-education-statistics-common/src/components/Accordion.module.scss

```diff
@@ -21,3 +21,34 @@
 .section {
   page-break-inside: avoid;
 }
+
+@mixin accordion-section-expanded() {
+  display: block;
+  height: auto;
+  padding: 15px 0;
+  visibility: visible;
+}
+
+.accordion :global(.govuk-accordion__section-content) {
```

Comment:
Couldn't we just move all of these changes into a `AccordionSection.module.scss` and not use `:global`?